starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -29,41 +32,44 @@ def __init__(self, sigma, is_volume: bool=False, verbose=False, **kwargs) -> Non
         self.verbose = verbose
 
     @classmethod
-    def get_algorithm_name(cls):
+    def get_algorithm_name(cls) -> str:
         return "gaussian_low_pass"
 
     @classmethod
-    def add_arguments(cls, group_parser):
+    def add_arguments(cls, group_parser: argparse.ArgumentParser) -> None:
         group_parser.add_argument(
-            "--sigma", default=1, type=int, help="standard deviation of gaussian kernel")
+            "--sigma", default=1, type=float, help="standard deviation of gaussian kernel")
```

Comment:
how does this work with volumes?

Reply:
Also, I don't see any CLI parameterization for is_volume

Reply:
if a scalar is passed, it is broadcast across all axes of the volume, so valid inputs are a scalar or a tuple of length == len(array.shape)